README.md

```diff
@@ -106,7 +114,7 @@ See also [success stories](https://en.oceanbase.com/customer/home) and [Who is u
 
 # Contributing
 
-Contributions are highly appreciated. Read the [Contribute to OceanBase](https://github.com/oceanbase/oceanbase/wiki/Contribute-to-OceanBase) guide to getting started.
+Contributions are highly appreciated. Read the [development guide](docs/README.md) to getting started.
```

Comment:
Should we separate subdirectories by language, such as docs/en_us and docs/zh_cn?

Reply:
After consideration, I suggest that developers can read English, and it's hard to maintain two versions of docs especially keep them up-to-date.

We may provide multilingual docs for users, but perhaps a single version of the dev guide is the best trade-off given the audience and maintain burden.

Reply:
okey, i got it